Adds new source: Unsplash Daily and Unsplash Random

Ashish285 opened the pull request:
Two new sources are added, unsplash_daily and unsplash_random. Unsplash_random pulls 1920x1080 image. Unsplash daily can have variable resolution.

Ashish285 committed:
Adds unsplash random and daily source

Ashish285 committed:
Merge branch 'master' of https://github.com/Ashish285/freshpaper

Ashish285 committed:
Syntax fix

Ashish285 committed:
updated readme

guptarohit commented:
@Ashish285 Thanks man for adding one more source! :+1:
Here, I noticed (two days) that the daily wallpaper here is in portrait mode, which won't be fit for desktops, however unsplash_random is super fine.

Ashish285 commented:
Yeah, i also noticed the vertical wallpapers. I will then remove the unsplash_daily wallpaper and only keep unsplash_random.

guptarohit commented on freshpaper.py:
```diff
@@ -215,10 +215,85 @@ def download_image_nasa(download_dir, image_extension="jpg"):
+    :param image_extension: directory where to download the image
+    :return: downloaded image path
+    """
+    # mkt(s) HIN, EN-IN
```
we don't need this comment, it was for bing only

Ashish285 replied:
OK, removed

Ashish285 committed:
Changes:
1.) Removed unsplash_daily as the wallpapers were portrait, not fit for desktop wallpapers.
2.) Added a check on NASA APOD as sometimes the Astronomy Picture of the Day is not a picture but a video.
